src/utils/searchAndSort.js

```diff
@@ -108,48 +108,47 @@ export function sortByStatus(list, statusList, activitiesList) {
 }
 
 export function sortList(sortValue, list, statusList = '', activitiesList = '') {
-	if (sortValue !== 'none') {
```

Comment:
This only looks like a lot of changed lines, because I removed an if and the indentation of all nested lines changed accordingly.